js/utils.js

```diff
@@ -140,43 +143,56 @@ export const deleteErrors = (evt) => {
 };
 
 // Синхронизация значений селектов
-export const setValue = (evt) => {
+export function setValue (evt) {
 
   timeFields.forEach((timeField) => {
     timeField.value = evt.target.value;
   });
 
+}
+
+// Сбрасываем маркер в начальное состояние
+export const resetMarker = () => {
+  mainPin.setLatLng(
+    BASE_COORDS
+  );
 };
 
-// Инактивация элементов
-export const disableInterface = () => {
+// Добавляет координаты в поле адреса
+export const setAddressInput = (lat,lng) => {
```

Comment:
```suggestion
const setAddressInput = (lat,lng) => {
```

можно не экспортировать, тк ф-я используется только внутри утилс